app/controllers/admin/scoreboards_controller.rb

```diff
@@ -42,7 +42,7 @@ def destroy
   private
 
     def scoreboard_params
-      params.require(:scoreboard).permit(:name, :description, :academic_year_calendar_id, :public)
+      params.require(:scoreboard).permit(:name, :name_en, :name_cy, :description, :academic_year_calendar_id, :public)
```

Comment:
Small thing - could t_params be used here instead?
```
translated_params = t_params(Scoreboard.mobility_attributes)
params.require(:scoreboard).permit(:name, translated_params, :description, :academic_year_calendar_id, :public)

```
 Also, is :name required too as well as :name_en and :name_cy?

Reply:
:+1: will look at using t_params :)

Reply:
👍